BillingApp/BillingInfo/templates/test_show_all.html

```diff
@@ -1,53 +1,28 @@
-<!DOCTYPE html>
-<head>
-    <meta charset="UTF-8">
-    <title>TEST</title>
-    <style>
-        body {font-family:微軟正黑體!important;}
-        .button {
-            width: 110px;
-            height:40px;
-            background-color:#3695FA;
-            border-radius: 5px;
-            color: white; 
-            font-size: 16px;
-        }
-        table { 
-            border:1px solid #D3D3D3; 
-            font-size:16px; 
-            text-align:center;
-            border-collapse:collapse;
-        } 
-        th { 
-            background-color: #808080;
-            padding:10px;
-            border:2px solid #D3D3D3;
-            color:#fff;
-        } 
-        td { 
-            border:2px solid #D3D3D3;
-            padding:10px;
-        }
-    </style>
-</head>
+{% extends 'base.html' %}
+{% block content %}
 
 <div>
     <div style="display: flex; justify-content: center;"><h1>所有花費</div>
 <body style="display: flex; justify-content: center;">
     <center>
        <input type="button" class='button' value="新增一筆帳" onclick="location.href='/add_billing/'" >
-       <input type="button" class='button' value="查詢紀錄" onclick="location.href='/query/'" >
+       <button type="button" class='button'" >查詢記錄</button>
        <input type="button" class='button' value="登出" onclick="location.href='/user/logout/'" >
     </center>
     <br>
+    <center>
+        <form action="/query_one/" method="POST"> {% csrf_token %}
+        <div style="display:none" id="p1">請輸入日期：</p><p>Date: <input type="text" id="datepicker" name="one_date"><input type="submit" value="查詢"></p>
+        </form></div>
+    </center>
     <table>
         <tr>
             <th>項目</th>
             <th>花費</th>
             <th>時間</th>
             <th>刪除</th>
             </tr>
-    <form action="./del/" method="POST"> {% csrf_token %}
+    <form action="/del/" method="POST"> {% csrf_token %}
```

Comment:
use named url
like `{% url 'del' %}`, see [this](https://docs.djangoproject.com/en/3.0/topics/http/urls/#reversing-namespaced-urls)